clients/common/rocblas_gentest.py

```diff
@@ -196,17 +196,30 @@ def setdefaults(test):
     # These are only for dynamic defaults
     # TODO: This should be ideally moved to YAML file, with eval'd expressions.
 
-    if all([x in test for x in ('M', 'incx', 'strideScale')]) and test['function']=='ger_strided_batched':
-        test.setdefault('stride_x', int(test['M'] * abs(test['incx']) *
-                                    test['strideScale']))
+    if test['function']=='swap_strided_batched':    
```

Comment:
Should be written as `if test['function'] in ('swap_strided_batched'):` so that more functions can be added later.

Reply:
@daineAMD was the camel caps a convention to indicate an non-argument.   Can unify the naming for this meta argument.

Reply:
@mahmoodw introduced strideScale I believe. I don't think there's a specific reason for it to be camel case, we can probably change it to stride_scale so it matches the typical naming conventions we use.

Reply:
Please go ahead as in daineAMD's comment and use snake_case rather than camelCase

Reply:
sure should add something to indicate it is not directly an argument, ??? meta_ prefix

Reply:
> sure should add something to indicate it is not directly an argument, ??? meta_ prefix

Probably not, because there are dozens of YAML objects which are not arguments.

If you want to indicate it is a "pseudo-argument", you could prepend it with `pseudo_`. This would only apply to YAML objects which are used to create arguments, not to the dozens of existing YAML objects which are not arguments.